js-service/src/cacheRouter.ts

```
@@ -0,0 +1,43 @@
+  const { key } = req.body;
+  exec(`redis-cli DEL ${key}`, (error, stdout) => {
+    if (error) {
+      return res.status(500).json({ error: error.message });
+    }
+    res.json({ result: stdout.trim() });
+  });
+});
+
+router.post('/cache/render', (req, res) => {
+  const { template } = req.body;
+  const rendered = eval(template);
```

Biome (2.4.16)

[error] eval() exposes to security risks and performance issues.

(lint/security/noGlobalEval)